packages/store/src/schemas/AddressArray.sol

```diff
@@ -48,7 +48,6 @@ library AddressArray_ {
 
   /**
    * Push an element to the addresses array
-   * TODO: this is super inefficient right now, need to add support for pushing to arrays to the store core library to avoid reading/writing the entire array
```

Comment:
Curious to see if we have gas tests for before/after this change to see how much more efficient this method is compared to the previous.

Reply:
The real change would be when a separate event is added. Right now there's still a redundant full storage read

Reply:
there should be a gasreport for these changes too though (there're some for push too), wonder why it didn't get added